s3tests_boto3/functional/test_s3select.py

```diff
@@ -1162,3 +1281,36 @@ def test_bool_cast_expressions():
     res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select count(*) from s3object where cast(_1 as int) != 0 ;')).replace("\n","")
 
     s3select_assert_result( res_s3select_cast, res_s3select )
+
+@attr('s3select')
+def test_progress_expressions():
+
+    csv_obj = create_random_csv_object(1000000,10)
+
+    csv_obj_name = get_random_string()
+    bucket_name = "test"
+    upload_csv_object(bucket_name,csv_obj_name,csv_obj)
+
+    res_s3select_progress = run_s3select_with_progress(bucket_name,csv_obj_name,"select sum(int(_1)) from s3object;")
+
+    bytes_scanned = res_s3select_progress['Details']['BytesScanned']
+
+    s3select_assert_result( bytes_scanned, 4194304 )
```

Comment:
as mentioned, not good coverage (only the first response, which is always the same)

Reply:
ok